messages/context.json

```diff
@@ -106,38 +106,64 @@
   "pages.editor.configuration.tag.subcategory": "pages.editor.configuration.tag.subcategory",
   "pages.editor.configuration.tag.template": "pages.editor.configuration.tag.template",
   "pages.editor.container.editpath.label": "pages.editor.container.editpath.label",
+  "pages.editor.store.button.back.title": "Back to editor",
```

Comment:
Got it from rebasing with next. 

> Shouldn't all values be exactly the same as their respective keys

Actually, they should be what the translator would read to help them understand what they are translating, so they could be anything... We use this convention because it's easier, but they could be empty strings as well 🤔

Reply:
Got it! Since we're not using any particular context, perhaps it'd be better to just follow the convention, right?